packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -74,53 +75,53 @@ export type AsyncData<Data, Error> = _AsyncData<Data, Error> & Promise<_AsyncDat
 
 export function useAsyncData<
   ResT,
-  DataE = Error,
+  NuxtErrorDataT = any,
```

Comment:
what about defaulting to `unknown`?

Reply:
Sure. Should we consider defaulting to `unknown` directly in h3 (https://github.com/unjs/h3/blob/main/src/error.ts) as well?

Reply:
I would also recommend that but I know @pi0 may have a different timetable or opinion on this one.